docs/developer.rst

```diff
@@ -104,8 +104,7 @@ Then be sure to pass the tests::
 
 When contributing to pycodestyle, please observe our `Code of Conduct`_.
 
-To run the Travis CI tests, you'll have to create a build the pycodestyle
-package using tox::
+To run the tests, use tox::
```

Comment:
What about "To run the tests the same way Travis CI runs them, use tox::"

I think that captures the intent of the original awkward phrasing.